src/konnektor/network_planners/generators/_abstract_network_generator.py

```diff
@@ -24,32 +24,34 @@ def __init__(
         self,
         mappers: Union[AtomMapper, list[AtomMapper]],
         scorer,
+        ## TODO: rename this to network_algorithm?
```

Comment:
We'd have to consider what this means to the class names everywhere else, but I agree that it would make sense.

Reply:
issue opened [here](https://github.com/OpenFreeEnergy/konnektor/issues/100)